packages/utils/telemetry-utils/src/test/errorLogging.spec.ts

```diff
@@ -673,7 +673,7 @@ describe("normalizeError", () => {
 });
 
 /** Create an error missing errorType that will not be recognized as a valid Fluid error */
-const createExternalError = (m) => new LoggingError(m);
+const createExternalError = (m) => new Error(m);
 
 /** Create a simple valid Fluid error */
 const createTestError = (m) =>
```

Comment:
nit: While you're here can you fix the indentation on this?  (the  next line should be indented I think)

Reply:
Sounds good. I have added the changes in commit #a1d49538.